src/client/debugger/jupyter/kernelDebugAdapter.ts

```diff
@@ -496,31 +498,46 @@ export class KernelDebugAdapter implements DebugAdapter, IKernelDebugAdapter, ID
         await this.dumpCell(cell.document.uri.toString());
 
         if (this.configuration.__mode === KernelDebugMode.RunByLine) {
-            const initialBreakpoint: DebugProtocol.SourceBreakpoint = {
```

Comment:
We should re-open https://github.com/microsoft/vscode-jupyter/issues/7251 or create a new issue to track the work around, else I do'nt see how we'll end up removing the work around.
I.e. theres no issue to track this work.

Reply:
I'll create a new one

Reply:
#7339